File: t-tasks/src/main/java/com/teo/ttasks/widget/configure/TasksWidgetConfigureView.java
package com.teo.ttasks.widget.configure;

import com.teo.ttasks.data.model.TTaskList;
import com.teo.ttasks.ui.base.MvpView;

import java.util.List;

interface TasksWidgetConfigureView extends MvpView {

    void onTaskListsLoaded(List<TTaskList> taskLists);

    void onTaskListsLoadError();
}

File: t-tasks/src/main/java/com/teo/ttasks/widget/configure/TasksWidgetConfigureView.kt
package com.teo.ttasks.widget.configure

import com.teo.ttasks.data.model.TaskList
import com.teo.ttasks.ui.base.MvpView

internal interface TasksWidgetConfigureView : MvpView {

    fun onTaskListsLoaded(taskLists: List<TaskList>)

    fun onTaskListsLoadError()
}
